packages/xsnap/src/xsnap.c

```diff
@@ -308,10 +322,15 @@ int main(int argc, char* argv[])
 	{
 		char done = 0;
 		while (!done) {
+			// By default, use the infinite meter.
+      gxCurrentMeter = 0;
```

Comment:
as dckc pointed out, this doesn't match the tab/spaces convention of the original, this looks weirdly indented